test/dd/test_package.cpp

```diff
@@ -477,41 +478,55 @@ TEST(DDPackageTest, SerializationErrors) {
   EXPECT_THROW(dd->deserialize<dd::mNode>(ss), std::runtime_error);
 }
 
-TEST(DDPackageTest, Extend) {
-  auto dd = std::make_unique<dd::Package<>>(4);
+TEST(DDPackageTest, TestConsistency) {
+  auto dd = std::make_unique<dd::Package<>>(2);
 
-  auto id = dd->makeIdent(3);
-  EXPECT_EQ(id.p->v, 2);
-  EXPECT_EQ(id.p->e[0], id.p->e[3]);
-  EXPECT_EQ(id.p->e[1], id.p->e[2]);
-  EXPECT_TRUE(id.p->isIdentity());
+  auto hGate = dd->makeGateDD(dd::H_MAT, 2, 1);
+  auto cxGate = dd->makeGateDD(dd::X_MAT, 2, 1_pc, 0);
+  auto zeroState = dd->makeZeroState(2);
 
-  auto ext = dd->extend(id, 0, 1);
-  EXPECT_EQ(ext.p->v, 3);
-  EXPECT_EQ(ext.p->e[0], ext.p->e[3]);
-  EXPECT_EQ(ext.p->e[1], ext.p->e[2]);
-  EXPECT_TRUE(ext.p->isIdentity());
-}
+  auto bellMatrix = dd->multiply(cxGate, hGate);
+  dd->incRef(bellMatrix);
+  auto local = dd->isLocallyConsistent(bellMatrix);
+  EXPECT_TRUE(local);
+  auto global = dd->isGloballyConsistent(bellMatrix);
+  EXPECT_TRUE(global);
+  dd->debugnode(bellMatrix.p);
 
-TEST(DDPackageTest, Identity) {
-  auto dd = std::make_unique<dd::Package<>>(4);
+  auto bellState = dd->multiply(bellMatrix, zeroState);
+  dd->incRef(bellState);
+  local = dd->isLocallyConsistent(bellState);
+  EXPECT_TRUE(local);
+  global = dd->isGloballyConsistent(bellState);
+  EXPECT_TRUE(global);
+  dd->debugnode(bellState.p);
+}
 
-  EXPECT_TRUE(dd->makeIdent(0).isOneTerminal());
+TEST(DDPackageTest, TestLocalInconsistency) {
+  auto dd = std::make_unique<dd::Package<>>(3);
 
-  auto id3 = dd->makeIdent(3);
-  EXPECT_EQ(dd->makeIdent(0, 2), id3);
-  const auto& table = dd->getIdentityTable();
-  EXPECT_NE(table[2].p, nullptr);
+  auto hGate = dd->makeGateDD(dd::H_MAT, 2, 0);
+  auto cxGate = dd->makeGateDD(dd::X_MAT, 2, 0_pc, 1);
+  auto zeroState = dd->makeZeroState(2);
 
-  auto id2 = dd->makeIdent(0, 1); // should be found in idTable
-  EXPECT_EQ(dd->makeIdent(2), id2);
+  auto bellState = dd->multiply(dd->multiply(cxGate, hGate), zeroState);
+  auto local = dd->isLocallyConsistent(bellState);
+  EXPECT_FALSE(local);
+  bellState.p->ref = 1;
+  local = dd->isLocallyConsistent(bellState);
+  EXPECT_FALSE(local);
+  bellState.p->ref = 0;
+  dd->incRef(bellState);
 
-  auto id4 = dd->makeIdent(0, 3); // should use id3 and extend it
-  EXPECT_EQ(dd->makeIdent(0, 3), id4);
-  EXPECT_NE(table[3].p, nullptr);
+  bellState.p->v = 2;
+  local = dd->isLocallyConsistent(bellState);
+  EXPECT_FALSE(local);
+  bellState.p->v = 1;
 
-  auto idCached = dd->makeIdent(4);
-  EXPECT_EQ(id4, idCached);
+  bellState.p->e[0].w.r->ref = 0;
+  local = dd->isLocallyConsistent(bellState);
+  EXPECT_FALSE(local);
+  bellState.p->e[0].w.r->ref = 1;
 }
```

Comment:
Something went wrong with the merge here in these sections.
I believe the consistency methods have been removed from the main branch and the identity tests can be removed as part of this PR. So, in the end, none of these lines should be here anymore.